### PR TITLE
Stale actions

### PR DESCRIPTION
**Pull request by NaveenKundeti-IBS**
added stale workflow

### CODE REVIEW

File: .github/workflows/stale.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Stale Pull Requests
+on:
+  # schedule:
+  #   - cron: '0 09 * * *'  
+  workflow_dispatch:
```
Comment (tomegathericon) [inline .github/workflows/stale.yaml]: @NaveenKundeti-IBS  : This key does not need any value?

Comment (NaveenKundeti-IBS) [reply .github/workflows/stale.yaml]: Sir, It needs a `cron` value to run the "stale" actions. Please suggest a time to run these actions. I'll uncomment the `schedule` event and update the workflow @tomegathericon

Comment (tomegathericon) [reply .github/workflows/stale.yaml]: Recommendations ? @NaveenKundeti-IBS

Comment (NaveenKundeti-IBS) [reply .github/workflows/stale.yaml]: Yes @tomegathericon sir, the recommended time to run the scheduled actions.

Comment (tomegathericon) [reply .github/workflows/stale.yaml]: I am asking for recommendations from you @NaveenKundeti-IBS

Comment (NaveenKundeti-IBS) [reply .github/workflows/stale.yaml]: I would recommend scheduling to run every day at 9:00 AM IST @tomegathericon sir

Comment (NaveenKundeti-IBS) [reply .github/workflows/stale.yaml]: @tomegathericon Sir, this is updated now!

**Review by tomegathericon — APPROVED** [review]
👍

**Pull request merged**